sites/upsun/src/changelog/_index.md

```diff
@@ -1,7 +1,24 @@
 ---
 title: "Changelog"
 weight: -14
-description: "This page is redirected to our [product portal](https://next.platform.sh/) with recently released features and services, what's currently being developed, and what's on the roadmap for the future."
+description: "We're always open to new and fresh ideas that can help improve our product. Learn how to make a feature request and engage with the {{% vendor/name %}} community here."
 ---
 
 {{% description %}}
+
+## Feature Requests
+We love to hear from you! Here’s how you can submit a feature request:
+
+### If you already have an Upsun account: 
+
+You can open a [support ticket](/learn/overview/get-support.html) directly from [your Console](https://console.platform.sh/). Simply describe your needs and our team will take it from there.
```

Comment:
```suggestion
You can open a [support ticket](/learn/overview/get-support.html) directly from [your Console](https://console.platform.sh/). Describe your needs and our team will take it from there.
```

Reply:
Value will keep failing on new PRs unless we follow this rule (and apply this change) or remove the rule from the tests.